scss/_patterns_pagination.scss

```diff
@@ -52,15 +52,15 @@
   [class*='p-icon'] {
     .p-pagination__link--next &,
     .p-pagination__link--previous & {
-      margin-left: $sph-inner--small;
-      margin-right: $sph-inner--small;
+      margin-left: $sph--small;
+      margin-right: $sph--small;
 
       &:first-child {
-        margin-left: -#{$sph-inner--small};
+        margin-left: -#{$sph--small};
       }
 
       &:last-child {
-        margin-right: -#{$sph-inner--small};
+        margin-right: -#{$sph--small};
```

Comment:
Is interpolation needed in these cases? There's an example of a negative variable in another file in this PR [here](https://github.com/canonical-web-and-design/vanilla-framework/pull/3971/files#diff-cb995e8a23b1a2dd7f10daff427c1a2298183ee0506d3ffd29fc0a62af5170fdR17) that doesn't use interpolation

Reply:
I don't know why GH won't allow me to add a suggested change here, but yes, this interpolation could be removed.